pypika/tests/test_tables.py

```diff
@@ -4,6 +4,7 @@
     Schema,
     Table,
 )
+from pypika.queries import make_tables
```

Comment:
this function is given an alias in `pypika/__init__.py` as `Tables`. Could you import it using that, please?

`from pypika import Tables`

Reply:
Sure. No problem. 

As you notice there is no any restriction creating many tables at the same time (i.e use the same alias only once or create many tables with the same name). I've supposed that pypika does not restrict anything about that.